### PR TITLE
heading-dependent-counters:0.1.0

### PR DESCRIPTION
**Pull request by jbirnick**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: Make counters depend on the heading counter.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

File: packages/preview/heading-dependent-counters/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[package]
+name = "heading-dependent-counters"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Johann Birnick <https://jbirnick.net>"]
+license = "MIT"
+description = "Make counters inherit from the heading counter."
+repository = "https://github.com/jbirnick/typst-rich-counters"
```
Comment (elegaanz) [inline packages/preview/heading-dependent-counters/0.1.0/typst.toml:8]: I think this should be updated or removed.

**Comment by elegaanz** [comment]
Also, the package name is bit too descriptive of what the package does, can you try to come up with something else please?

**Comment by jbirnick** [comment]
Hmm.. idk, do you have any ideas? :D

Is `simple-inherited-counters` fine?

**Comment by elegaanz** [comment]
Yes, that would work. :)

**Pull request closed without merging**